quiz.py

```diff
@@ -1,6 +1,64 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Matthew Daguanno
+Date: 2024-03-04
+Description: Asks 3 questions and gives 3 options to pick from for each.
+"""
+
+print("Multiple Choice Quiz Game")
+
+counter = 0
+
+# Question one
+q1_inp = input("1. What is the capital of France?")
+print("(a) Paris")
+print("(b) London")
+print("(c) Rome")
+
+if q1_inp == "a":
+    print("Correct!")
+    counter = counter + 1
+elif q1_inp == "b":
+    print("Incorrect.")
+elif q1_inp == "c":
+    print("Incorrect.")
```

Comment:
There is another way to handle two selection branches with identical actions. See notes on logical operators [here](https://github.com/STE-Technology/ICS3U-Unit-2-Control-Flow#logical-operators).